upgrade/UPGRADE-v9.1.3.md

```diff
@@ -1,6 +1,6 @@
-# [Upgrade from v9.1.2 to v9.1.3-dev](https://github.com/shopsys/shopsys/compare/v9.1.2...9.1)
+# [Upgrade from v9.1.2 to v9.1.3](https://github.com/shopsys/shopsys/compare/v9.1.2...9.1)
 
-This guide contains instructions to upgrade from version v9.1.2 to v9.1.3-dev.
+This guide contains instructions to upgrade from version v9.1.2 to v9.1.3.
```

Comment:
In this file there is several upgrade notes for changes not mentioned in changelog